core/src/main/scala/org/apache/spark/internal/config/UI.scala

```diff
@@ -102,7 +102,7 @@ private[spark] object UI {
   val UI_HEAP_HISTOGRAM_ENABLED = ConfigBuilder("spark.ui.heapHistogramEnabled")
     .version("3.5.0")
     .booleanConf
-    .createWithDefault(SystemUtils.isJavaVersionAtLeast(JavaVersion.JAVA_11))
+    .createWithDefault(true)
```

Comment:
I piggyback another change while I am here. I believe these are all instances to fix with `isJavaVersionAtLeast` ... if I didn't miss ..

Reply:
cc @dongjoon-hyun

Reply:
+1